sympy/assumptions/refine.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import print_function, division
 
+#from sympy.matrices import MatrixExpr
```

Comment:
It may be deleted if not needed

Reply:
But if we delete it, it doesn't pass a test.

Reply:
Travis tests were not passing because of some network error. Presence of an unimportant comment can do nothing to the tests.

Reply:
Sorry I got confused. I didn't realise that was a comment. Got it confused with something else.